truffle/src/com.oracle.truffle.api.test/src/com/oracle/truffle/api/test/polyglot/ContextSharingTest.java

```diff
@@ -320,6 +320,7 @@ public void testThreadCheck() throws InterruptedException, ExecutionException {
 
     @Registration(id = SINGLE_THREAD, name = SINGLE_THREAD)
     public static class SingleThreadedLanguage extends TruffleLanguage<Env> {
+        public static final String ID = SINGLE_THREAD;
```

Comment:
not used

Reply:
Fixed in 75e60df. No longer needed because we removed `org.graalvm.polyglot.Context.Builder` API and its associated test.